index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <header class="header-container">
+        <nav class="nav-item1">My logo</nav>
+        <nav class="nav-item2"></nav>
+        <nav class="nav-item3">
+            <img src="images/toggle.png" alt="toggle">
+        </nav>
+    </header>
+    <div class="content">
+        <h1 class="intro">I’m Toyosi Glad to see you!</h1>
+        <p class="intro1">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h3 class="intro2">LET’S CONNECT</h3>
+        <ul class="social">
+            <li class="social-icon">
+                <img src="images/twitter.png" alt="twitter">
+            </li>
+            <li class="social-icon">
+                <img src="images/linkedin.png" alt="linkedin">
+            </li>
+            <li class="social-icon">
+                <img src="images/m.png" alt="m">
+            </li>
+            <li class="social-icon">
+                <img src="images/github.png" alt="github">
+            </li>
+            <li class="social-icon">
+                <img src="images/fingers.png" alt="fingers">
+            </li>
+        </ul>
```

Comment:
- [ ] Great job using ul's to layout the icons. However, the icons should be a link so that when they're clicked they open the respective web page.
```
<ul>
  <li> <a></a></li>
</ul>
```